drizzlepac/hapsequencer.py

```diff
@@ -579,7 +579,7 @@ def run_hap_processing(input_filename, diagnostic_mode=False, input_custom_pars_
             log.warning("")
             log.warning("There are no viable direct images in any Total Data Product for this visit. No processing can be done.")
             log.warning("No SVM processing is done for the Grism/Prism data - no SVM output products are generated.")
-            sys.exit(0)
+            sys.exit(65)
```

Comment:
If it can be avoided, "magic numbers" should not be used in the source code.  Use **analyze.Ret_code.NO_VIABLE_DATA.value**.

Reply:
fixed